docs/src/main/paradox/cluster-sharding.md

```diff
@@ -0,0 +1,92 @@
+---
+project.description: Alpakka Kafka provides a module to use Kafka with Akka Cluster External Sharding.
+---
+# Akka Cluster Sharding
+
+Akka Cluster allows the user to use an external shard allocation strategy in order to give the user more control over
+how many shards are created and what cluster nodes they are assigned to. If you consume Kafka messages into your 
+Akka Cluster application then it's possible to run an Alpakka Kafka Consumer on each cluster node and co-locate Kafka
+partitions with Akka Cluster shards. When partitions and shards are co-located together then there is less chance
+that a message must be transmitted over the network by the Akka Cluster Shard Coordinator to a destination user sharded
+entity.
+
+@@project-info{ projectId="clusterSharding" }
+
+@@dependency [Maven,sbt,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-stream-kafka-cluster-sharding_$scala.binary.version$
+  version=$project.version$
+}
+
+This module contains an Akka extension called `KafkaClusterSharding`.
+There are two steps required to setup the cluster sharding module.
+
+* A `MessageExtractor` to route Kafka consumed messages to the correct Akka Cluster shard and user entity
+* A `RebalanceListener` to update the external shard allocation at runtime when Kafka Consumer Group rebalances occur
+
+@@@ note
+
+A complete example of using this module exists in an [`akka/akka-sample`](https://github.com/akka/akka-samples) project 
+called [`akka-sample-kafka-sharding`](https://github.com/akka/akka-samples/tree/2.6/akka-sample-kafka-to-sharding-scala).  
+It's a self-contained example that can run on a developer's laptop.
+
+@@@
+
+## Dependency
+
+This module directly depends on `akka-cluster-sharding-typed` version 2.6.3 or later.
+
+@@dependency [Maven,sbt,Gradle] {
+  symbol=AkkaVersion
+  value=$akka.version26$
+  group=com.typesafe.akka
+  artifact=akka-cluster-sharding-typed_$scala.binary.version$
+  version=AkkaVersion
```

Comment:
Have just one `@@dependency` directive on the page.

Reply:
I dropped this second `@@dependency` directive and moved the note about requiring 2.6.3 above the first directive.